src/main/frontend/components/page.cljs

```diff
@@ -777,16 +791,16 @@
            [:div.l.flex.items-center
             [:div.actions-wrap
              (ui/button
-               [(ui/icon "trash") (t :delete)]
-               :on-click (fn []
-                           (let [selected (filter (fn [[_ v]] v) @*checks)
-                                 selected (and (seq selected)
-                                               (into #{} (for [[k _] selected] k)))]
-                             (when-let [pages (and selected (filter #(contains? selected (:block/idx %)) @*results))]
-                               (state/set-modal! (batch-delete-dialog pages false #(do
-                                                                                     (reset! *checks nil)
-                                                                                     (refresh-pages)))))))
-               :small? true)]
+              [(ui/icon "trash") (t :delete)]
+              :on-click (fn []
+                          (let [selected (filter (fn [[_ v]] v) @*checks)
+                                selected (and (seq selected)
+                                              (into #{} (for [[k _] selected] k)))]
```

Comment:
we could re-write above 3lines to 
`(into #{} (comp (filter second) (map first)) @*checks)`